docs/en/operation/Troubleshooting.md

```diff
@@ -249,6 +258,8 @@ installation or `spark/conf/` if you customize this file for Spark) has the foll
 </configuration>
 ```
 
+See the doc page for your specific compute framework for detailed setup instructions.
```

Comment:
(nit)add example link here?

Reply:
I didn't want to enumerate all compute options like spark here and hive here and presto here. We don't have a URL like `https://docs.alluxio.io/os/user/stable/en/compute/` (have to specify spark or hive etc). :(